src/software/git/github/Automatic-deployment.md

```diff
@@ -0,0 +1,94 @@
+# 自动部署 Github Pages 和服务器
+
+## 生成配置 GitHub Actions Token
+
+- 生成 Token ：<https://github.com/settings/tokens>
+
+- 或者可以点击，头像 → Settings → Developer settings → Personal access tokens → Generate new Token
+
+- Token 名称填写 Token， Select scopes 勾选 repo，然后滚动到网页最下面点击提交按钮。生成了 Token，复制保存(该 Token 只显示一次，忘记了就再生成)
+
+- 配置到项目的 Secrets 中：进入项目 → Settings → Secrets → New secret
+    
+    - Name ：建议和刚才生成 Token 保持一致
+    - Value ：为刚才生成的 Token
+
+- 除了配置 Token，还要配置服务器的 HOST、 USERNAME、 PASSWORD、 PORT，这些配置在 GitHub 把项目部署到服务器的时候使用到

```

Comment:
Again, chinese symbols are full width, no space is needed, but a half width english symbol should before a space (unless the last one on a line)